src/PositionVelocityTime.jl

```diff
@@ -23,24 +27,56 @@ module PositionVelocityTime
         carrier_phase::CP = 0
     end
 
+    """
+    DOP, stores the Dilution of Precision values
+    """
+    @with_kw mutable struct DOP
+        GDOP::Union{Nothing, Float64} = nothing
+        PDOP::Union{Nothing, Float64} = nothing
+        VDOP::Union{Nothing, Float64} = nothing
+        HDOP::Union{Nothing, Float64} = nothing
+        TDOP::Union{Nothing, Float64} = nothing
+    end
+
+
     """
     PVTSolution, with used sats and el and az.
     """
     @with_kw mutable struct PVTSolution
```

Comment:
Again: Does it need to be mutable?
A non-mutable struct would be more efficient and also makes it easier to reason about.
It is better to create a new `PVTSolution` with the updated values.
`@with_kw` makes this really easy.

Reply:
You are right, I'll change that.